app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] Kindly add a method that returns the 3 most recent posts.

Reply:
Well resolved 👍🏽